src/richie/plugins/lti_consumer/models.py

```diff
@@ -0,0 +1,21 @@
+"""
+LTI consumer plugin models
+"""
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from cms.models.pluginmodel import CMSPlugin
+
+from richie.apps.courses.models import Course
+
+
+class LTIConsumer(CMSPlugin):
+    """
+    LTI consumer plugin model.
+    """
+
+    url = models.URLField(_("LTI url"))
+    course = models.ForeignKey(to=Course, on_delete=models.PROTECT)
+
```

Comment:
Note that you also need a field to point to one of the LTI configurations in settings.
We will need to force this field for some placeholders (see what we did to force CKEditor configuration on the SimpleText plugin) so that a placeholder can only receive an LTI plugin targetting Marsha and not some other LTI service that we don't want to see here...

Reply:
It might be ok now